rules/scripts/bitstreams_workspace.py

```diff
@@ -35,17 +35,37 @@
 XMLNS = {'': 'http://doc.s3.amazonaws.com/2006-03-01'}
 # Manifest schema directory
 MANIFESTS_DIR = os.path.dirname(__file__) if __file__ else os.path.dirname(sys.argv[0])
+# Required designs
+KNOWN_DESIGNS = {
+    "chip_earlgrey_cw310": {
+        "bitstream": "@//hw/bitstream/vivado:fpga_cw310_test_rom",
+        "rom_mmi": "@//hw/bitstream/vivado:rom_mmi",
+        "otp_mmi": "@//hw/bitstream/vivado:otp_mmi",
+    },
```

Comment:
This is going to be a bit of a pain to ensure it's up-to-date, though. CI doesn't have a good way to test this ...yet?

I could construct something with a blank bitstreams cache and test that the loading phase works. :thinking:

Reply:
Done.